R/layer.r

```diff
@@ -229,6 +229,15 @@ Layer <- ggproto("Layer", NULL,
     evaled <- lapply(aesthetics, rlang::eval_tidy, data = data)
     evaled <- compact(evaled)
 
+    nondata_cols <- check_nondata_cols(evaled)
+    if (length(nondata_cols) > 0) {
+      msg <- paste0(
+        "Aesthetics must be valid data columns: `", rlang::as_label(aesthetics[[nondata_cols]]),
+        "`. Did you mistype the name of a data column or forget to add stat()?"
```

Comment:
I would remove the backticks around the aesthetics, because it'll look strange if you have more than one.

Reply:
Backticks removed